tracer/src/Datadog.Trace/Generated/net461/Datadog.Trace.SourceGenerators/ConfigurationKeysGenerator/ConfigurationKeys2.DataStreamsMonitoring.g.cs

```diff
@@ -0,0 +1,31 @@
+﻿// <copyright company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+// <auto-generated/>
+
+#nullable enable
+
+// This file is auto-generated from supported-configurations.json and supported-configurations-docs.yaml
+// Do not edit this file directly. The source generator will regenerate it on build.
+// NOTE: If you remove keys/products from the JSON, run 'dotnet clean' and remove old generated files.
+namespace Datadog.Trace.Configuration;
+
+internal static partial class ConfigurationKeys2
+{
+    internal static class DataStreamsMonitoring
+    {
+        /// <summary>
+        /// Enables data streams monitoring support
+        /// <see cref="Datadog.Trace.Configuration.TracerSettings.IsDataStreamsMonitoringEnabled"/>
```

Comment:
Semantically I think the `<see>` section should come after the `<summary>` section